packages/neutrino-preset-web/index.js

```diff
@@ -143,7 +146,12 @@ module.exports = (neutrino, opts = {}) => {
         .use(optimize.ModuleConcatenationPlugin);
     })
     .when(neutrino.options.command === 'build', (config) => {
-      neutrino.use(clean, { paths: [neutrino.options.output] });
+      if (options.clean) {
```

Comment:
For consistency with the rest of the middleware, could you switch this to `.when`? Also, I don't think we need to include the `output` condition here since you have that set above in the options with `paths: [neutrino.options.clean]`:

```js
config.when(options.clean, () => neutrino.use(clean, options.clean));
```